src/libs/actions/TaxRate.ts

```diff
@@ -0,0 +1,174 @@
+import type {OnyxCollection} from 'react-native-onyx';
+import Onyx from 'react-native-onyx';
+import * as API from '@libs/API';
+import {WRITE_COMMANDS} from '@libs/API/types';
+import * as ErrorUtils from '@libs/ErrorUtils';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+import type {Policy} from '@src/types/onyx';
+import type {OnyxData} from '@src/types/onyx/Request';
+
+let allPolicies: OnyxCollection<Policy>;
+Onyx.connect({
+    key: ONYXKEYS.COLLECTION.POLICY,
+    waitForCollectionCallback: true,
+    callback: (value) => (allPolicies = value),
+});
+
+function setPolicyCustomTaxName(policyID: string, customTaxName: string) {
```

Comment:
Same comment as in the other PR. I feel like we can leave these commands in the policy.ts file as we dont have separate actions files for Tags/ Categories/ Members etc

If we decide to break it to multiple actions files later, it can be separate initiative

Reply:
Done!